common/browser_options_test.go

```diff
@@ -262,4 +263,41 @@ func TestBrowserLaunchOptionsParse(t *testing.T) {
 			}
 		})
 	}
+
+	// will be passed to NewLaunchOptions to set onCloud to false.
+	const onCloud = false
```

Comment:
Very minor, so you can ignore this suggestion.

```suggestion
	const notOnCloud = false
```

I think this reads better when it's used:

```go
wantOpts: NewLaunchOptions(notOnCloud),
```

vs

```go
wantOpts: NewLaunchOptions(onCloud),
```

Reply:
Or maybe set `onCloud` to `true`, and then:

```go
wantOpts: NewLaunchOptions(!onCloud),
```